docs/docs/services/email.md

```diff
@@ -4,11 +4,43 @@ title: email
 parent: Services
 ---
 
-# data
+# email
 {: .no_toc }
 
 ## Summary
 The email service deploys the lambdas, SNS topics, and Configuration Sets needed to send email.
 
 ## Detail
-AWS SES is an account-wide service for basic sending and receiving of email.  By creating lambdas to build the emails and sending the email with a branch-specific configuration set, we can follow the events of email sending and take action based on those events.
\ No newline at end of file
+AWS SES is an account-wide service for basic sending and receiving of email.  By creating lambdas to build the emails and sending the email with a branch-specific configuration set, we can follow the events of email sending and take action based on those events.
+
+### SSM Parameters
+The workflow will not successfully deploy unless the emailAddressLookup object is defined:
+
+Named {project}/default/emailAddressLookup or {project}/{stage}/emailAddressLookup
+    {   
+        "sourceEmail":"\"CMS MACPro no-reply\" <spa-reply@cms.hhs.gov>",
+        "osgEmail":"\"OSG\" <mako.cms.osg@gmail.com>",
+        "chipInbox":"\"CHIP Inbox\" <mako.cms.osg@gmail.com>",
+        "chipCcList":"\"CHIP CC 1\" <k.grue@theta-llc.com>;\"CHIP CC 2\" <k.grue.stateadmn@gmail.com>",
+        "dpoEmail":"\"DPO Action\" <mako.cms.osg@gmail.com>",
+        "dmcoEmail":"\"DMCO Action\" <mako.cms.osg@gmail.com>",
+        "dhcbsooEmail":"\"DHCBSOO Action\" <mako.cms.osg@gmail.com>"
+    }
+
+These values are set during deployment as environment variables on the lambda.  You can edit these values in the AWS Console on the Lambda configuration tab.
```

Comment:
When I researched doing this, I read is that there is a charge per lookup for secrets.  I tried using the parameter store, which is free to lookup, and was dissatisfied with the result.  I also don't like adding an asynchronous dependency on something that changes very rarely.  I feel like, if later we add some way for a CMS admin to update the emails (the first reason I could come up with for a lookup), it's pretty obvious we will need to update this lambda to support, so I guess I wasn't worrying about it, since I decided I didn't like it :P .

That being said, you guys are the architects and I know using the secrets can be done if that is what you want.